### PR TITLE
Update Particular.Packaging to latest and add required LICENSE file

### PR DESCRIPTION
**Pull request by mauroservienti**

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+By accessing NServiceBus code here, you are agreeing to the following licensing terms.
```
Comment (bording) [inline LICENSE.md]: This doesn't look like the right content. It should be the from RepoStandards.

Comment (mauroservienti) [reply LICENSE.md]: I’ll copy pasted from another repo, to get it from RepoStandards can I simply copy it from that repo?

Comment (bording) [reply LICENSE.md]: Yes, that's fine.

**Comment by bording** [comment]
Looking at the build failure, there might some additional tweak needed to the GitVersion.yml to ignore a commit.

I'll take a look.

File: LICENSE.md (hunk 1 above)
Comment (mauroservienti) [reply LICENSE.md]: Copied from RepoStandards (https://github.com/Particular/NServiceBus.AmazonSQS/pull/308/commits/121a14891d2f0e2c42f0765d8860007fec3abb36)

**Comment by andreasohlund** [comment]
@mikeminutillo seems to have the same issue with the linux tests not reporting :(

**Pull request merged**